backend/scripts/nlp/lambda_function.py

```diff
@@ -0,0 +1,56 @@
+""" This is a copy of the Lambda function that was uploaded to AWS, it is not intended to run locally
+and it will not work without modification"""
+
+import json
+import nltk
+import requests
+nltk.download('punkt')
+
+
+
+
+
+def lambda_handler(event=None, context=None):
+    sentence = "Who am I"
+    # sentence = event["sentence"]
+    
+    print(sentence)
+    tokens = nltk.word_tokenize(sentence.lower())
+    tokenlist = ''
+    firstword = tokens[0]
+    print(firstword)
```

Comment:
remove all print statements for lambda function, they will not be helpful if running on lambda